geoengine/workflow.py

```diff
@@ -232,6 +235,28 @@ def plot_chart(self, bbox: QueryRectangle) -> VegaLite:
 
         return VegaLite(vega_spec)
 
+    def get_array(self, bbox: QueryRectangle) -> np.ndarray:
+        '''
+        Query a workflow and return the raster result as a numpy array
+        '''
+        session = get_session()
+
+        # TODO: properly build CRS string for bbox
```

Comment:
what is missing here?

Reply:
really understand the schema and be able to output non-epsg urn strings

Reply:
mh i guess we would just replace epsg with sr-org?

Reply:
yeah but there are other ogc specific formats, e.g. one for WGS84 with other axis order etc. I just wrote this todo to make clear that this is not the final way in which we build that string